src/frontend/screens/YourTeam/CollaboratorInfo.tsx

```diff
@@ -136,10 +144,24 @@ export const CollaboratorInfo: NativeNavigationComponent<
               <MaterialDesignIcons size={size} color={color} name="export" />
             )}
             onPress={() => {
+              if (isLastMember) {
+                // TODO: Navigate to interstitial warning screen for last member when it is made
+                console.log(
+                  'Last member: Show warning that project will be empty',
+                );
+                return;
+              }
+              if (isLastCoordinator) {
+                // TODO: Navigate to interstitial warning screen for last coordinator when it is made
+                console.log(
+                  'Last coordinator: Show warning that no coordinators will remain',
+                );
+                return;
+              }
               navigation.navigate('LeaveProject');
             }}
           />
-        ) : ownRoleIsCoordinator ? (
+        ) : ownRoleIsCoordinator && !isOwnDevice ? (
```

Comment:
That comment didnt show up in my review...but `&& !isOwnDevice ` is redundant as were already checking for `isOwnDevice` above